src/plugin.ts

```diff
@@ -26,6 +26,7 @@ export interface RecycleBinOptions {
    * will continue "in the back". **This value should be smaller than `maxItemsInRequest`**
    * otherwise it has no effect. Defaults to `5`. */
   maxItemsWithResponse: number;
+  recycleItemPostHook: PostHookHandlerType<string>;
```

Comment:
I think you can set it as optional, so you will also need to apply the posthook only if it is defined.